docs/pages/attributesJSON.rst

```diff
@@ -46,9 +46,125 @@ Below are the supported JSON tags for Physics Manager Attributes templates, and
 	- list of strings
 	- A list of locations to query for supported object files that should be available to be loaded into the world.
 
+
+`Scene Dataset Attributes`_
+===========================
+Scene Dataset Attributes describe the names and file locations of all the assets and configuration files that make up a Scene Dataset. They are designed as a kind of index for a scene dataset, organizing the links to all the constituent components of the dataset into appropriate thematic categories, as well as providing an easy mechanism for customizing any of the configurations, all in a single file. All file paths referenced in this configuration should be relative to the directory where this Scene Dataset configuration resides.
```

Comment:
And what is a scene dataset?

Reply:
What we need is a glossary -- basically a list of terms we use and what they mean.